omero/sysadmins/unix/server-centos7-ice36.rst

```diff
@@ -119,6 +119,14 @@ Configure the database and the location of the data directory:
     :start-after: #end-copy-omeroscript
     :end-before: #end-step04
 
+Weaker ciphers like ADH are disabled by default in OpenSSL 1.1+.
```

Comment:
Although the statement is correct, it does not match the fact that CentOS ships `openssl` 1.0.2.
Should we also include here a sentence mentioning that TLS 1.0/1.1 are EOL and getting dropped in the JDK packages?